pyshelf/utils.py

```diff
@@ -83,3 +83,41 @@ def validate_bucket_config(config):
 
     if len(name_list) != len(unique_list):
         raise ValueError("Error in bucket config. Overlapping bucket names and reference names.")
+
+
+def validate_bucket(config, bucket_name):
+    """
+        Validates the bucket_name passed in with the request is a valid bucket_name
+        and a ref name is used if one is configured.
+
+        Args:
+            config(dict)
+            bucket_name(string)
+
+        Returns:
+            boolean: whether bucket is found and correct name is used.
+    """
+    for bucket in config["buckets"]:
+        if bucket_name == bucket["name"] and bucket.get("referenceName") is None:
```

Comment:
Have the two returns leaves a bad taste in my mouth.  Can we instead solve this by making shorter variable names?

```
refName = bucket.get("referenceName")
name = bucket["name"]

if (bucket_name == name and refName is None) or bucket_name == refName:

# or on separate lines

if (bucket_name == name and refName is None)
    or bucket_name == refName:
```

I think some variant of that is valid